### ISSUE TITLE
Weird crash while loading.

### ISSUE THREAD

**Issue by 63502** [rwtema/extrautilities_old_issues#1010]
I have no idea why, but apparently extrautils2-1.10.2-alpha-1.1.0a causes this crash to happen on startup. the crash is not linked to Random Things, as when i remove RT, Tinkers Construct replaces it. I only know that if I use the previous version of Extra utils, everything works fine and the game loads normally.

Crash report: https://gist.github.com/63502/d7947f3d416ad244c606100df18712eb

**Comment by ghost** [comment]
Have you tried taking away Optifine? It doesn't seem a problem with those specific mods that you say.
I'm having a crash too with **extrautils2-1.10.2-alpha-1.1.0a** and **OptiFine_1.10.2_HD_U_C3** (even 1.10.2_HD_U_C2), but the old **extrautils2-1.10.2-alpha-1.1.0** has no problem whatsoever with Optifine.

**Comment by DatrixTHLK** [comment]
I can verify this. If you have tinkers construct or Mantle API then Optifine and Mantle will crash your game because anything using mantle will not cooperate with Optifine and cause a crash, even though Optifine isn't the **real** issue. 

Minecraft 1.10.2
Forge: 12.18.1.2095
Extrautils2: 1.1.0a

I am having the same issue with the newest version of ExtraUtils2 1.1.0a the older version or 1.0.1 worked just fine without crashing. Downgrading until issue is resolved.

Crash Report: https://gist.github.com/DatrixTHLK/64a31b4d1b97aa23278e3b5da63d90b9

**Comment by 63502** [comment]
I did the same

**Comment by kurokuno** [comment]
still kinda sucks hope its fixed soon given the new version has so much content quantum quarry deep dark all kinds of crazy stuff =D and before the default remove optifine is an option keep in mind on my 980 16gbs of ram i5-4690 3.5 ghz i get 40-60 fps very unstable with no vsync in my 1.10.2 world with optifine i have not droped under 200 frames and average about 290 with all the default optifine settings

**Comment by DatrixTHLK** [comment]
@kurokuno  Have you tried a mod called BetterFPS? It could be another idea, though at the moment I use both Optifine and BetterFPS. Nevermind BetterFPS doesn't help that much in my situation.

The unfortunate part is that some of us don't have beefy computers and have very unstable frame issues. It could be that some mods aren't very well optimized with the game. Which is why Optifine has helped a bit to stabilize that issue. Unfortunately, those who want to try out mantle mods are restricted to not being able to use their mods because it doesn't like Optifine being installed (One reason I will never download Tinkers Construct. Armour Reborn is still new but needs more armor variants similar to Tinkers Construct to be a viable alternative). In the past Optifine was said to have many rendering issues that weren't well liked with the community (or so I've heard), but have since changed and is working just fine. 

Right now I'm going to see what FPS I get without Optifine installed and how many frames I get with Optifine installed and will bring back the results.

Minecraft 1.10.2:
**EDIT (11:30am):** Without Optifine I experience lag and the game will glitch and freeze for split seconds.  FPS: 20-55 max

With Optifine I experience none of those issues or are barely noticeable. FPS: 100-130 max and these are on default for some of the settings but I put some on fast because it gives more FPS 

I also have 8gbs of ram with AMD FX 8350 Eight core Processor 4.34Ghz

**Comment by 63502** [comment]
Yeah, I need optifine to get any lv of fps above 30-40 too.

Btw, version 1.1.0b does not fix this.

**Comment by DatrixTHLK** [comment]
Same here...

https://gist.github.com/DatrixTHLK/28629dcc4d6e65b4b25e9ec3759bf013

Extra Utilities: 1.1.0b 

I sometimes wonder if we should show these rendering crash reports to Optifine's mod author...

**Comment by 63502** [comment]
I'll wait to do that until this bug gets looked at from the extrautils end first.

Still the same with 1.1.0c
[Crash Log](https://gist.github.com/63502/c528895152241131849d9ef8af9b0836)

@rwtema Help? Please?

**Comment by DatrixTHLK** [comment]
Yes, good idea. In that case I will hang back at 1.0.1 until then.

**Comment by AngelCry** [comment]
one more with same issue here - crash on startup with any EU 1.1.0. everything same - i also have TC and cant play without Optifine, so for now i stay with EU 1.0.1

**Comment by xChitose** [comment]
I've actually sent Optifine a message before asking about something in modded MC and I haven't gotten a reply yet.

I sent it over a week ago, but it was just a question, I'm sure they're busy with other things. 

@DatrixTHLK

**Comment by DatrixTHLK** [comment]
Well I know they are making an update to Optifine as shown in the preview C4. As I've said before I'll wait. Let me know if you get anything back from Optifine. @xChitose Could you possibly reference the issue mentioned on Optifine's issue page?

@63502 Looks like RWTema won't help with this situation as mentioned in his readme file. Maybe something will come of the message xChitose sent to Optifine and maybe it will get fixed. For now I would uggest staying at 1.0.1 until issue is completely resolved. Removing Optifine is **NOT** an option, at least for those of us with unstable framerate.

**Comment by DatrixTHLK** [comment]
This is supposedly fixed in the in the newest build of Optifine (Preview C4). I've heard a few people say it fixed what happened with Extra Utilities. Let me know what you guys think.

**Comment by AngelCry** [comment]
well, i already tried C4 pre few days ago, and got same crash, so is been not fixed in any way. but i not sure if they replace preview build from time to time and how often if so. now i see in changelog:
- fixed Forge compatibility (Extra Utilities 2 crash)

so, is worth to recheck. anyway, this issue does not hurt me so much - i can live without quantum quarry, because RFTools builder looks as good alternative :)

**Comment by AngelCry** [comment]
ok, i have good news. just downloaded latest C4 pre and checked it with EU 1.1.0c and everything loaded correctly, without crashes, so looks like is finally fixed. good FPS for everyone :)

**Comment by DatrixTHLK** [comment]
Yeah I also noticed that the pre C4 had a bug fix to it and that is more than likely why it is working for you now. @63502 @Zekralyx @kurokuno @xChitose  The issue has been fixed with latest Optifine Preview build. No more need to bother RWTema about this. At least for EU2 and TC, they cooperate. Baubles is another issue (or so I've heard). 

Next time we have rendering issues we should take the problem straight to Optifine. I've learned my lesson.
